liquidator/constants.js

```diff
@@ -0,0 +1,9 @@
+// As defined in 1inch
+const ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```

Comment:
Why doesn't 1inch use 0x0....0?

Reply:
It uses both, `0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE` was chosen to be more explicit